packages/quantic/force-app/main/default/lwc/quanticTimeframeFacet/__tests__/quanticTimeframeFacet.test.js

```diff
@@ -127,6 +191,31 @@ function mockSuccessfulHeadlessInitialization() {
   };
 }
 
+function mockErroneousHeadlessInitialization() {
+  // @ts-ignore
```

Comment:
Why do we need those?

is it simply because the `mockHeadlessLoader` doesn't have a proper type that contains a `initializeWithHeadless` ?

Reply:
no it's because of the following error: Cannot assign to 'initializeWithHeadless' because it is a read-only property.